### DISCUSSION TITLE
How to set user status to unpaid?

### DISCUSSION THREAD

**Discussion by petermarks12** [Glench/ExtPay#147] [Q&A]
I am working one extension where user may need to pay again and again so I need to know is there any way to change the status of paid customer to unpaid so that he can pay again?

I am providing access to the user on an hourly or daily basis.  Once the time is over, I would like him to pay again.  I am not able to figure out this part and also I could not find much related to this in the documentation.  Any help or ideas will be appreciated.

**Comment by Glench** [comment]
This is not currently possible with ExtensionPay.
